neural_compressor/torch/algorithms/weight_only/rtn.py

```diff
@@ -73,7 +74,14 @@ def rtn_quantize(
     Returns:
         model: fake quantized torch module
     """
-    device = "cpu"
+    device_name = kwargs.pop("device", "auto")
+    runtime_accelerator = auto_detect_accelerator(device_name)
+    device = runtime_accelerator.name()
```

Comment:
How about moving this device checking to `environ.py` so that all algos can call the result directly?

Reply:
Thanks, moved it into environ.py in [fixed UTs](https://github.com/intel/neural-compressor/pull/1622/commits/0186e67aaeb20412c8b0e70c0db46e8131701f15).